src/components/BigDate/BigDate.js

```diff
@@ -1,30 +1,34 @@
 import React from 'react';
-import { withStyles } from '@material-ui/core/styles';
+import { makeStyles } from '@material-ui/core/styles';
 import Typography from '@material-ui/core/Typography';
 import PropTypes from 'prop-types';
 import dayjs from 'dayjs';
 
-const styles = theme => ({
+const useStyles = makeStyles(theme => ({
 	container: {
 		display: 'flex',
 		flexDirection: 'column',
 		alignItems: 'center',
 		width: 'fit-content'
 	},
+	month: {
+		fontWeight: theme.typography.fontWeightBold
+	},
 	day: {
 		fontSize: theme.typography.fontSize * 24 / 16,
 		fontWeight: theme.typography.fontWeightLight,
 		lineHeight: 0.8
 	}
-});
+}));
 
-function BigDate({ date, classes }) {
+function BigDate({ date }) {
+	const classes = useStyles();
 	const dateObj = dayjs(date);
 	const month = dateObj.format('MMM');
 	const day = dateObj.format('D');
 	return (
 		<div className={classes.container}>
```

Comment:
This could be a [`<time>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time).